rllib/algorithms/ppo/tf/ppo_tf_policy_rlm.py

```diff
@@ -131,7 +129,9 @@ def loss(
             mean_vf_unclipped_loss = tf.reduce_mean(vf_loss)
         # Ignore the value function.
         else:
-            vf_loss_clipped = mean_vf_loss = tf.constant(0.0)
+            mean_vf_unclipped_loss = (
+                value_fn_out
+            ) = vf_loss_clipped = mean_vf_loss = tf.constant(0.0)
```

Comment:
dude, why don't we write this out in a simpler way. this looks weird.
```
mean_vf_unclipped_loss = tf.constant(0.0)
value_fn_out = tf.constant(0.0)
...
```

Reply:
oh lint picked this weird formatting :) ok noted. I'll fix it.